src/Files.App/UserControls/Widgets/DrivesWidget.xaml

```diff
@@ -9,7 +9,7 @@
 	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
 	mc:Ignorable="d">
 
-	<Grid>
+	<Grid Padding="16,0,16,0">
```

Comment:
?

Reply:
I think it's better to fix this on the ListView as opposed to making changes on the Expander

Reply:
then I have to set negative margins on the ListView.

Reply:
Isn't that what we did before?